<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CxMeanGrayMeasForm</class>
 <widget class="QWidget" name="CxMeanGrayMeasForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mean Gray Measurement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="btnNewMeas">
     <property name="text">
      <string>New measurement</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnRemove">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QTableWidget" name="tableResults">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Idx</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mean Gray</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>StDev</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Min</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Max</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Area</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Width</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Height</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnClear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="CxExportToolButton" name="btnExport">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="btnGraph">
     <property name="text">
      <string>Time Graph...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CxExportToolButton</class>
   <extends>QToolButton</extends>
   <header>ExportToolButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
